extensions/wikia/EditPageLayout/EditPageLayoutController.class.php

```diff
@@ -206,6 +210,19 @@ public function executeEditPage() {
 			? wfMessage( 'editpagelayout-notificationsLink-none' )->escaped()
 			: wfMessage( 'editpagelayout-notificationsLink', count( $this->notices ) )->parse();
 
+		try {
+			$templateType = ( new TemplateClassificationService() )
```

Comment:
Please consider if the call to this service should depend on wgEnableTemplateClassificationExt variable.